bin/common.bash

```diff
@@ -268,3 +274,95 @@ ck8s_kubespray_version_check(){
         exit 1
     fi
 }
+with_kubeconfig() {
+    kubeconfig="${1}"
+    shift
+
+    if [ ! -f "${kubeconfig}" ]; then
+      log_error "ERROR: Kubeconfig not found: ${kubeconfig}"
+      exit 1
+    fi
+
+    if grep -F -q 'sops:' "${kubeconfig}" || \
+        grep -F -q '"sops":' "${kubeconfig}" || \
+        grep -F -q '[sops]' "${kubeconfig}" || \
+        grep -F -q 'sops_version=' "${kubeconfig}"; then
```

Comment:
Are all these variants needed?

Reply:
Yes, this method was actually taken from the apps repo ( https://github.com/elastisys/compliantkubernetes-apps/blob/main/bin/common.bash#L552 )

Reply:
Seems like it has been updated there, so I'll probably refresh it here as well